src/SFA.DAS.ApprenticeCommitments.Web/Pages/Shared/UserNameExtension.cs

```diff
@@ -7,6 +7,9 @@ namespace SAF.DAS.ApprenticeCommitments.Web
 {
     public static class UserNameExtension
     {
+        public static Claim? ApprenticeIdClaim(this ClaimsPrincipal user)
+            => user.Claims.FirstOrDefault(c => c.Type == "apprentice_id");
+
```

Comment:
"apprentice_id" is used a few times, Would it be better as a constant

Reply:
I wasn't sure if it would, so I made the change to see and it prompted me to clean up some more.  Yes I'd say it is better.